packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -82,19 +92,52 @@ class _AndroidViewState extends State<AndroidView> {
     );
   }
 
-  @override
-  void initState() {
-    super.initState();
+  void _initializeOnce() {
+    if (_initialized) {
+      return;
+    }
+    _initialized = true;
+    assert(widget.layoutDirection != null || debugCheckHasDirectionality(context));
+    _layoutDirection = widget.layoutDirection ?? Directionality.of(context);
     _createNewAndroidView();
   }
 
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    _initializeOnce();
+
+    final TextDirection newLayoutDirection = _findLayoutDirection();
+    final bool didChangeLayoutDirection = _layoutDirection != newLayoutDirection;
+    _layoutDirection = newLayoutDirection;
+
+    if (didChangeLayoutDirection) {
+      _controller.setLayoutDirection(_layoutDirection);
```

Comment:
Maybe leave a code comment here in case we turn unawaited_futures linter on later

Reply:
done